projects/canopy/src/lib/forms/sort-code/sort-code.notes.ts

```diff
@@ -0,0 +1,45 @@
+export const notes = `
+# Sort Code Component
+
+## Purpose
+Provides a form component that can be used to capture a sort code in three individual input fields. The result, when used in a reactive form, is a concatenation of the three values into one string e.g. '204060'. The overall field is only valid if all three input fields are populated with 2 digits.
+
+## Usage
+Import the Sort Code Module into your application:
+
+~~~js
+@NgModule({
+  ...
+  imports: [LgSortCodeModule],
+})
+~~~
+
+and in your HTML:
+
+~~~html
+<lg-sort-code formControlName="sortCode">
+  Sort Code
+  <lg-hint id="hintId">Must be 6 digits long</lg-hint>
+  <lg-validation id="errorId" *ngIf="isControlInvalid(sortCode, validationForm)">
+    Your sort code should be a 6 digit number
+  </lg-validation>
+</lg-sort-code>
+~~~
+
+## Inputs
+
+### LgSortCodeComponent
+| Name | Description | Type | Default | Required |
+|------|-------------|:----:|:-----:|:-----:|
+| \`\`value\`\` | Existing 6 digit sort code string that will prepopulate the three input fields appropriately | string | '' | No |
+| \`\`disabled\`\` | Sets the three inner input fields to disbaled if \`\`true\`\` | boolean | \`\`false\`\` | No |
+| \`\`firstId\`\` | HTML ID attribute for the first number input, auto generated if not provided | string | \`\`lg-input-date-\${nextUniqueId++}\`\` | No |
```

Comment:
`nextUniqueId++` is incorrect here I think - it should be  just `id`?

Reply:
I think `nextUniqueId++` is right, as that's the name of the variable that generates it. Unless I'm missing something? I actually also just removed the `++` too, as per your other comment 🙂

Reply:
I think we've used that convention elsewhere, just to imply that a default is provided, but it's not a static default.